src/expense.ts

```diff
@@ -0,0 +1,98 @@
+import { collection, doc, addDoc, deleteDoc, updateDoc, Timestamp } from 'firebase/firestore';
+
+import db from './firebase';
+
+export interface ExpenseDataFirestore {
+  title: string;
+  date: Timestamp;
+  amount: number;
+}
+
+export interface ExpenseDataForm {
+  title: string;
+  date: Date;
+  amount: string;
+}
+
+export class Expense {
+  title: string;
+  date: Date;
+  amount: number;
+  id: string;
+
+  constructor(expense: ExpenseDataFirestore, id: string) {
+    this.title = expense.title;
+    this.date = expense.date.toDate();
+    this.amount = expense.amount;
+    this.id = id;
+  }
+
+  // Adds expense data to "expenses" collection on firestore
```

Comment:
the documentation in the merge request description is exemplary, but it will be lost as soon as we merge this MR. can we somehow put the relevant information here? what standards are out there for documenting javascript code?